### PR TITLE
Manual  done

### PR DESCRIPTION
**Pull request by LukasJonsson99**
Pictures on the cards is still not perfect

### CODE REVIEW

File: src/components/info/ActivityInfo.tsx [hunk 1]
```diff
@@ -1,11 +1,74 @@
...
+      contentContainerStyle={{alignItems: 'center'}}>
+      <Headline>Manual för aktivitetsvyn</Headline> 
+        <Text style={styles.text}>
+          I aktivitesvyn kan du både registrera aktiviteter och se vilka aktiviteter som du redan har registrerat. 
+          För att komma till aktivitetsvyn så navigerar du till plusstecknet som finns längst till vänster i navigationsbaren.
+        </Text>
+        <Subheading>Startsidan</Subheading>
+        <Text style={styles.text}>
+          När du har navigerat till aktivitetsvyn möts du av en sida med 12 ikoner och 3 menyknappar som finns längst ner på skärmen.
+          De 12 ikonerna är dina favoritikoner och dessa kan ändras under iställningar. Menyknappen som är längst till vänster
```
Comment (Dolvur) [inline src/components/info/ActivityInfo.tsx]:
```suggestion
          De 12 ikonerna är dina favoritikoner och dessa kan ändras under inställningar. Menyknappen som är längst till vänster
```

File: src/components/info/ActivityInfo.tsx [hunk 2]
```diff
@@ -1,11 +1,74 @@
...
+          innehåller massor av ikoner som är där om du känner att dina favoritikoner inte räcker till. Menyknappen som ligger i mitten
```
Comment (Dolvur) [inline src/components/info/ActivityInfo.tsx]:
```suggestion
          tar upp en lista på resten av ikonerna. Menyknappen som ligger i mitten
```

File: src/components/info/ActivityInfo.tsx [hunk 3]
```diff
@@ -1,11 +1,74 @@
...
+          navigerar dig till en sida där du kan registrera planerade aktiviteter, mer om det senare. Menyknappen som är längst till höger 
+          navigerar dig till en sida där du kan se en historik på dina registrerade aktiviteter, mer om det senare.
```
Comment (Dolvur) [inline src/components/info/ActivityInfo.tsx]:
```suggestion
          navigerar dig till en sida där du kan registrera planerade aktiviteter. Menyknappen som är längst till höger 
          navigerar dig till en sida där du kan se en historik på dina registrerade aktiviteter. 
```

File: src/components/info/ActivityInfo.tsx [hunk 4]
```diff
@@ -1,11 +1,74 @@
...
+        </Text>
+        <Image
+          style={styles.image}
+          source={require('../../images/activityStartscreen.png')}
+        />
+        <Subheading>Registrera en aktivitet</Subheading>
+        <Text style={styles.text}>
+          För att registrera en aktivitet så trycker du bara på en ikon som bäst representerar den aktivitet som du har gjort. 
+          Det tar dig till en vy där du kan bestämma dagen och tiden som aktiviteten genomfördes. Här kan du också, om du vill,
+          beskriva aktiviteten mer detaljerat eller också koppla den till en aktivitet som du har definierat i värderingsvyn.
```
Comment (Dolvur) [inline src/components/info/ActivityInfo.tsx]:
```suggestion
          Det tar dig till en vy där du kan bestämma dagen och tiden som aktiviteten genomfördes. Här kan du 
          beskriva aktiviteten mer detaljerat eller koppla den till en aktivitet som du har definierat i värderingsvyn.
```

File: src/components/info/ActivityInfo.tsx [hunk 5]
```diff
@@ -1,11 +1,74 @@
...
+          Därefter graderar du hur meningsfull och underhållande aktiviteten har varit.
+        </Text>
+        <Image
+          style={styles.image}
+          source={require('../../images/activityRegistrator.png')}
+        />
+        <Subheading>Historik</Subheading>
+        <Text style={styles.text}>
+          I historiken finns alla aktiviteter som du har registrerat där du på varje aktivitet också kan se hur du graderade den. 
+          Här kan du också betygsätta hela din dag. För att se historiken för en annan dag trycker du bara på kalendern längst
```
Comment (Dolvur) [inline src/components/info/ActivityInfo.tsx]:
```suggestion
          Här kan du också betygsätta hela din dag. För att se historiken för en annan dag trycker du bara på datumet längst
```

File: src/components/info/ActivityInfo.tsx [hunk 6]
```diff
@@ -1,11 +1,74 @@
...
+        </Text>
+        <Image
+          style={styles.image}
+          source={require('../../images/activityHistory.png')}
+        />
+        <Subheading>Registrera planerade aktiviteter</Subheading>
+        <Text style={styles.text}>
+          För att registrera en aktivitet som du redan har planerat in så trycker du bara på menyknappen som ligger i mitten på
+          startsidan. Där finns en vy för att se dagens inplanerade aktiviteter och bara genom att trycka på dem så kan du
+          registrera dem.
```
Comment (Dolvur) [inline src/components/info/ActivityInfo.tsx]:
```suggestion
          För att registrera en aktivitet som du redan har planerat in så trycker du på menyknappen som ligger i mitten på
          startsidan. Där finns en vy för att se dagens inplanerade aktiviteter och genom att trycka på en så kan du
          registrera den.
```

**Review by Dolvur — APPROVED** [review]
Looks good!

Before you merge you should localize the information so that it works in English as well.

**Review by malin-mallan — COMMENTED** [review]
Really good text in swedish!

Good with borders around the images but they are a bit uneven in thinkness.

This colud be fixed when the titels are merged: The titel should be on the header instead

File: src/components/info/ValuesInfo.tsx [hunk 7]
```diff
@@ -1,11 +1,81 @@
...
+    },
+    heading: {
+      fontSize: 30,
+    },
+    subheading: {
+      paddingTop: 15,
+      fontSize: 22,
+    },
+    
+  });
```
Comment (EMattfolk) [inline src/components/info/ValuesInfo.tsx]: This could probably be moved to a separate file

File: src/language/en.tsx [hunk 8]
```diff
@@ -49,9 +49,9 @@ const en = {
     activityWeekHistoryTitleDay: 'Today\'s history',
     activityWeekHistoryTitleWeek: 'Activity history',
 
-    informationCardActivityscreen: 'Activityscreen',
-    informationCardValuesscreen: 'Valuesscreen',
-    informationCardCalenderscreen: 'Calenderscreen',
+    informationCardActivityscreen: 'Activityinfo',
+    informationCardValuesscreen: 'Valuesinfo',
+    informationCardCalenderscreen: 'Calenderinfo',
```
Comment (EMattfolk) [inline src/language/en.tsx]: I think the titles needs to be changed a bit.
```suggestion
    informationCardActivityscreen: 'Activities',
    informationCardValuesscreen: 'Values',
    informationCardCalenderscreen: 'Calendar',
```

File: src/language/sv.tsx [hunk 9]
```diff
@@ -49,9 +49,9 @@ const sv = {
     activityWeekHistoryTitleDay: 'Dagens historik',
     activityWeekHistoryTitleWeek: 'Aktivitetshistorik',
 
-    informationCardActivityscreen: 'Aktivitetsvy',
-    informationCardValuesscreen: 'Värderingsvy',
-    informationCardCalenderscreen: 'Kalendervy',
+    informationCardActivityscreen: 'Aktivitetsinfo',
+    informationCardValuesscreen: 'Värderingsinfo',
+    informationCardCalenderscreen: 'Kalenderinfo',
```
Comment (EMattfolk) [inline src/language/sv.tsx]:
```suggestion
    informationCardActivityscreen: 'Aktiviteter',
    informationCardValuesscreen: 'Värderingar',
    informationCardCalenderscreen: 'Kalender',
```

File: src/components/info/CalenderInfo.tsx [hunk 10]
```diff
@@ -1,11 +1,62 @@
 import React from 'react';
-import { View } from 'react-native';
-import { Text } from 'react-native-paper';
+import { View, Image, StyleSheet } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import { Text, Headline, Subheading, Paragraph } from 'react-native-paper';
 
 export const CalenderInfo = () => {
```
Comment (EMattfolk) [inline src/components/info/CalenderInfo.tsx]:
```suggestion
export const CalendarInfo = () => {
```
The file name also needs changing

**Review by EMattfolk — APPROVED** [review]
It looks good.

Found some smaller things.

**Comment by Neathan** [comment]
Think i found one of the reasons why the type checks fail. The languages files seem to not match on this branch and because the `Language` type is defined as `typeof en` there is a type mismatch when you try to assign `const dict = languages[settings.language];` to a provider which is created with type of `sv`.

The solution would be to update the language files so that they match each other, and in the long run we probably want to change how we define the `Language` type.

Edit: I've gotten some information that they in fact do contain the information but in the wrong order, thus giving the same result.

File: src/language/sv.tsx [hunk 11]
```diff
@@ -80,12 +82,98 @@ const sv = {
         AddEntryView: 'Lägg till aktivitet',
```
Comment (Neathan) [inline src/language/sv.tsx:82]: Are these capitalized to match the internal names of the screens?

Comment (EMattfolk) [reply src/language/sv.tsx:82]: Yes

**Pull request merged**